packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -584,20 +557,21 @@ describe('Ceramic API', () => {
       expect(Object.keys(docs).length).toEqual(6)
     })
 
-    it('can load docs for array of multiqueries even if docid or path throws error', async () => {
-      const queries = [
-        {
-          docId: docA.id,
-          paths: ['/b/d', '/notExistDocId']
-        },
-        {
-          docId: notExistDocId,
-          paths: ['/e/f' , '/d']
-        }
-      ]
-      const docs = await ceramic.multiQuery(queries, 1000)
-      expect(Object.keys(docs).length).toEqual(3)
-    })
+    // FIXME 754 Problematic test for #754
```

Comment:
How did the change from this PR make this test worse?  It looks totally unrelated to me?

Reply:
Totally unrelated feature-wise. At one point I anticipated dropping "--runInBand", and this test still consumer resources after "ceramic.close".

For future reference: multi-query timeout is a shallow one. The timeout does not propagate to `Document.load`, so it hangs till IPFS timeout expires.

Reply:
Going to put it back in a few.

Reply:
CC @zachferland about multi-query timeout.

Reply:
Uncommented back.